fhir-odoo/pom.xml

```diff
@@ -37,7 +37,7 @@
     <maven.compiler.source>17</maven.compiler.source>
     <maven.compiler.target>17</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <odoo-java-api.version>3.1.8</odoo-java-api.version>
+    <odoo-java-api.version>3.3.5</odoo-java-api.version>
```

Comment:
We will need to publish this version to our Mekom Nexus repo, as it's not available in any other online repo.